packages/eslint-config-sheriff/src/getStorybookConfig.ts

```diff
@@ -0,0 +1,16 @@
+import storybook from 'eslint-plugin-storybook';
+import tseslint from 'typescript-eslint';
+import type { TSESLint } from '@typescript-eslint/utils';
+
+export const getStorybookConfig = (): TSESLint.FlatConfig.ConfigArray => {
+  return tseslint.config({
+    extends: [
+      storybook.configs['flat/recommended'],
+      storybook.configs['flat/csf'],
+    ],
+    rules: {
+      'import/no-default-export': 0,
```

Comment:
@AndreaPontrandolfo, eslint-plugin-import is applied _after_ storybook, so this unfortunately doesn't do anything.
I don't see any harm in moving this to the end though.

Reply:
> eslint-plugin-import is applied after storybook

Why do you say so? I'm pretty sure this is not the case.

Reply:
> > eslint-plugin-import is applied after storybook
> 
> Why do you say so? I'm pretty sure this is not the case.

Huh. Yeah, maybe not. It looked like it was behind it. Somehow though, that patch still doesn't get applied.
I'll go debug some more.

Reply:
Oh, actually it's because the default is false. Based on this comment:

> it should always be checked on by default, we shouldn't even check the package.json

I'd assumed it would be on by default. Happy to see Sheriff get leaner though!

Reply:
Yeah, Storybook support should always be `true` by default.
Is this not the case for you? Then it might be a bug.

Reply:
No, it's always `false` by default.

Reply:
@lishaduck oh ok, i get it now.

```
  if (userConfigChoices.storybook) {
    exportableConfig.push(...getStorybookConfig());
  }
```

This should be:

```
  if (userConfigChoices.storybook !== false) {
    exportableConfig.push(...getStorybookConfig());
  }
```

Reply:
Yup!

Reply:
(I still don't see why you wouldn't want it opt-in like everything else, but you do you :))

Reply:
> (I still don't see why you wouldn't want it opt-in like everything else, but you do you :))

Because the Storybook plugin applies only to storybook files. I think it's more reasonable  to have it opt-out.

Reply:
Yeah, that's fair. I keep forgetting that ESLint filters down plugins like that. Still feels like it'd be slightly more performant with it off.